inc/class-sitemaps.php

```diff
@@ -79,12 +78,13 @@ public function register_sitemaps() {
 	public function setup_sitemaps() {
 		$sitemaps = $this->registry->get_sitemaps();
 
+		add_rewrite_tag( '%sub_type%', '([^?]+)' );
+
 		// Set up rewrites and rendering callbacks for each supported sitemap.
 		foreach ( $sitemaps as $sitemap ) {
-			add_rewrite_rule( $sitemap->route, 'index.php?sitemap=' . $sitemap->name . '&paged=$matches[1]', 'top' );
+			/** @noinspection PhpUndefinedMethodInspection */
+			add_rewrite_rule( $sitemap->route, $sitemap->rewrite_query(), 'top' );
```

Comment:
Is users using the base `Core_Sitemap_Provider::rewrite_query()` method here?

Reply:
Yes